src/gtk/webview_webkit.cpp

```diff
@@ -407,7 +407,16 @@ wxgtk_webview_webkit_context_menu(WebKitWebView *,
     if(webKitCtrl->IsContextMenuEnabled())
         return FALSE;
     else
-        return TRUE;
+    {
+        wxContextMenuEvent evtCtx(
+            wxEVT_CONTEXT_MENU,
+            webKitCtrl->GetId(),
+            webKitCtrl->ClientToScreen(wxDefaultPosition));
```

Comment:
We should use the `GdkEvent` parameter passed to the callback according to [the documentation](https://webkitgtk.org/reference/webkit2gtk/stable/WebKitWebView.html#WebKitWebView-context-menu) for the menu position.